### ISSUE TITLE
Starting from version 42 I get errors for all my files for the first line

### ISSUE THREAD

**Issue by HarelM** [gajus/eslint-plugin-jsdoc#1038]
The following is my project:
https://github.com/IsraelHikingMap/Site/tree/e854120951fe8f32057f2d6f2f3611d4c6f5dee6/IsraelHiking.Web
When I upgrade this plugin to version 42 or above I get error for all my files:

for example:
```
/Users/me/dev/Site/IsraelHiking.Web/src/application/services/traces.service.ts
  1:1  error  Definition for rule 'jsdoc/newline-after-description' was not found  jsdoc/newline-after-description
```
This is for the first line of every file in my project.
To resolve this currently I upgraded to version 41.1.2.
Thanks for this awesome plugin, let me know if there's anything else I can do.

To reproduce:
1. Clone the project
2. Go to IsraelHiking.Web
3. `npm i`
4. `npm i eslint-plugin-jsdoc@latest`
5. `npm run lint`

Thanks!!

**Comment by brettz9** [comment]
You should follow https://github.com/angular-eslint/angular-eslint/issues/1353 . We have the replacement info in the release notes, e.g., to use `jsdoc/tag-lines: ['any', {startLines: 0|1}]`

**Comment by HarelM** [comment]
Thanks for the super quick response! I'll follow the angular issue and see where it lands, I prefer to keep configuration as minimal as possible...

**Comment by davidmurdoch** [comment]
`jsdoc/tag-lines: ['any', {startLines: 0|1}]` isn't valid JS, JSON, or even TypeScript. What are we supposed to do?

**Comment by brettz9** [comment]
@davidmurdoch 

> `jsdoc/tag-lines: ['any', {startLines: 0|1}]` isn't valid JS, JSON, or even TypeScript. What are we supposed to do?

`|` is typically used to indicate a selection of choices. I.e., choose either 0 or 1. The rule should be quoted and added to the relevant `rules` block within your config, e.g.,

```js
rules: {
  'jsdoc/tag-lines': ['any', {startLines: 0}]
}
```

**Comment by davidmurdoch** [comment]
So if I had:

```
"jsdoc/newline-after-description": [
      "error"
],
```

then what should have now? Is there already a migration guide somewhere?

```
  "jsdoc/newline-after-description": "off",
  'jsdoc/tag-lines': ????????
```

**Comment by brettz9** [comment]
As per the [commit description](https://github.com/gajus/eslint-plugin-jsdoc/pull/1018/commits/799aa5f0f348c054ff07fd3643d01d320e4bc49d):

> Removes `jsdoc/newline-after-description` rule in favor of `jsdoc/tag-lines` with option `startLines: 0` for "never" and `startLines: 1` for "always". Defaults now to `startLines: 0`

The old behavior (in the `newline-after-description` rule) per that commit was to default to "always".

Therefore, you can use the following to preserve the old behavior:

```js
'jsdoc/tag-lines': ['error', {startLines: 1}]
```

**Comment by brettz9** [comment]
As per the [commit description](https://github.com/gajus/eslint-plugin-jsdoc/pull/1018/commits/799aa5f0f348c054ff07fd3643d01d320e4bc49d):

> Removes `jsdoc/newline-after-description` rule in favor of `jsdoc/tag-lines` with option `startLines: 0` for "never" and `startLines: 1` for "always". Defaults now to `startLines: 0`

The old behavior (in the `newline-after-description` rule) per that commit was to default to "always".

Therefore, you can use the following to preserve the old behavior:

```js
'jsdoc/tag-lines': ['error', 'any', {startLines: 1}]
```

**Comment by davidmurdoch** [comment]
Thanks! That commit description was not helpful for me. haha

Eslint's syntax, especially the shorthand, is just not noob friendly -- especially when all I wanted to do today was update TypeScript, and now I'm staring at 800+ lines of eslint with a dozen override sections all clobbering each other. 😅 

Thanks again!
